### PR TITLE
Removed docker network type `host`

### PR DESCRIPTION
**Pull request by mschultheiss83**
here are the changes from our live coding session

### CODE REVIEW

File: docker-compose-setup.yml [hunk 1]
```diff
@@ -37,4 +42,4 @@ services:
       - server
     network_mode: "host"
     profiles:
-      - donotstart
\ No newline at end of file
+      - donotstart1
```
Comment (TooAngel) [inline docker-compose-setup.yml:45]:
```suggestion
      - donotstart
```

File: utils/test.js [hunk 2]
```diff
@@ -6,6 +6,10 @@ const {setPassword, sleep, initServer, startServer, spawnBots, helpers, logConso
 
 const {cliPort, verbose, tickDuration, playerRoom, players, rooms, milestones} = require('./testConfig');
 
+// todo for local-testing
+// if your machine is slow try increment this
+const waitForConnection = 10;
```
Comment (TooAngel) [inline utils/test.js:11]: I would extract this into the `testConfig.js` as there are all the other configuration values aswell

**Review by TooAngel — COMMENTED** [review]
Looks good.

I would prefer to stick to the function definition without the space. `function test()`.

File: docker-compose-setup.yml (hunk 1 above)
Comment (mschultheiss83) [reply docker-compose-setup.yml]: sure

File: utils/test.js (hunk 2 above)
Comment (mschultheiss83) [reply utils/test.js]: done

**Comment by mschultheiss83** [comment]
wait before merge... 
the tests would not run through any more... 
screeps server is alwasy restarting

**Comment by mschultheiss83** [comment]
Should we add a WIP Tag? For this pull request or just close it? Or move this to a branch : local window test environment?

**Comment by TooAngel** [comment]
Let's keep it for now. I'll have a look later. I think this can only be fixed for all cases (windows, else, circleci)

**Review by mschultheiss83 — COMMENTED** [review]
commit is a little bit ugly
includes fixing typos ab80d429

**Review by TooAngel — APPROVED** [review]
Looks good

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**